### PR TITLE
Update changelog for 22.02 and archive info of some older releases [skip ci]

### PR DESCRIPTION
**Pull request by pxLi**
Signed-off-by: Peixin Li <pxli@nyu.edu>

The changelog grows big. For better readability and less generating time, I am trying to move some releases to archive folder

```
# created CHANGELOG_0.1_to_0.5.md of older releases
GITHUB_TOKEN=xxx scripts/generate-changelog --releases=0.1,0.2,0.3,0.4,0.4.1,0.5 --path=docs/archives/CHANGELOG_0.1_to_0.5.md

# Updated the CHANGELOG.md
GITHUB_TOKEN=xxx scripts/generate-changelog --releases=21.06,21.06.1,21.06.2,21.08,21.08.1,21.10,21.12,22.02
```
For rendered markdown files
https://github.com/pxLi/spark-rapids/blob/update-changelog-22.02/CHANGELOG.md
https://github.com/pxLi/spark-rapids/blob/update-changelog-22.02/docs/archives/CHANGELOG_0.1_to_0.5.md

please let me know if the location and naming of archived changelog looks good or not. Thanks!

### CODE REVIEW

**Comment by pxLi** [comment]
@sameerz can you help check if any incorrect labels/projects info of PR/issue? thx~

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,293 @@
...
+
+### Features
+|||
+|:---|:---|
+|[#4305](https://github.com/NVIDIA/spark-rapids/issues/4305)|[FEA] write nvidia tool wrappers to allow old YARN versions to work with MIG|
+|[#4360](https://github.com/NVIDIA/spark-rapids/issues/4360)|[FEA] Add explain api for Spark 2.X|
+|[#3541](https://github.com/NVIDIA/spark-rapids/issues/3541)|[FEA] Support max on single-level struct in aggregation context|
+|[#4238](https://github.com/NVIDIA/spark-rapids/issues/4238)|[FEA] Add a Spark 3.X Explain only mode to the plugin|
+|[#3761](https://github.com/NVIDIA/spark-rapids/issues/3761)|[FEA] Call the appropriate method when transitioning from Row<=>Col |
+|[#3949](https://github.com/NVIDIA/spark-rapids/issues/3949)|[Audit][SPARK-32709][SQL] Support writing Hive bucketed table (Parquet/ORC format with Hive hash)|
```
Comment (jlowe) [inline CHANGELOG.md]: We do not support writing a Hive bucketed table.  Either this issue should not have been closed or we need to fix the summary.

Comment (sameerz) [reply CHANGELOG.md]: I reopened issue #3949 and moved it to 22.04.

**Review by sameerz — COMMENTED** [review]
I updated the labels of some issues.  Can you please regenerate?  

Thanks for splitting the changes into a recent and older changelog file.

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
re-generate, we will do a final update right before the release

**Review by jlowe — APPROVED** [review]

**Comment by pxLi** [comment]
merge and will create an final update PR right before the 22.02 release

**Pull request merged**